project/baselitex-nexus.yaml

```diff
@@ -0,0 +1,20 @@
+srcs:
+  - third_party/vexriscv-verilog/VexRiscv_Full.v
+  - src/baselitex-nexus/baselitex_nexus.v
+top: top
+name: baselitex
+data:
+  - src/baselitex-nexus/mem.init
+  - src/baselitex-nexus/mem_1.init
+clocks:
+  PLL_0_P: 13.33333
+clock_aliases:
+  sys_clk:
+    - sys_clk
+    - sys_clk$glb_clk
+vendors:
+  lattice-nexus:
+    - lifcl-40
+#   - lifcl-17
```

Comment:
Maybe worth adding a comment on why this is not being run

Reply:
I made a change so that `nextpnr-nexus` uses its `router1` algorithm, which is what HPS uses, and now `oneblink` on `lifcl-17` also passes, so I've re-enabled it.

Reply:
It seems like we should have runs for both router algorithms?

Reply:
> It seems like we should have runs for both router algorithms?

Agreed, I have code for that.

*Currently* with router2, the design hits the infinite loop with design lifcl-17 and seed=1.   It's fine if the run times out and gets marked down as a failure, but I don't see any timeout / limit for each individual run.  @acomodi , will the `exhaust.py` script do the right thing and kill the one task, or will the never-ending job wreck the entire tool-perf run?

Reply:
At the moment the job will be never-ending if enters in a loop. We might add a feature to kill a job once a time limit is reached, to avoid infinite loops